### PR TITLE
little change to make reading more smoothly

### PR DESCRIPTION
**Pull request by W4anD0eR96**
.

### CODE REVIEW

File: zh.perl6intro.adoc [hunk 1]
```diff
@@ -983,7 +983,7 @@ unless $clean-shoes {
 
 === with
 
-`with` 的用法 `if` 相同, 不过 `with` 会检查变量是否定义。 +
+`with` 的用法 `if` 相同, 不过 `with` 会检查变量是否已经定义。 +
```
Comment (wenjie1991) [inline zh.perl6intro.adoc]: There should be a "和".
> `with` 的用法和 `if` 相同
or
> `with` 和 `if` 用法相同

Comment (W4anD0eR96) [inline zh.perl6intro.adoc]: should i now close this pull request and create a new?

Comment (W4anD0eR96) [inline zh.perl6intro.adoc]: it was updated

**Comment by wenjie1991** [comment]
Thanks ：）

**Pull request merged**